### ISSUE TITLE
Plone 5 support?

### ISSUE THREAD

**Issue by merpdotcom** [collective/Products.Ploneboard#49]
Are there any workarounds or plans to make Ploneboard work with Plone 5? Os is there an alternate Forum add-on for Plone 5? This is mission critical for most of my sites (over a dozen completely different communities/sites).
Thanks!

**Comment by djay** [comment]
@merpdotcom you might want to ask this on community.plone.org since its not really a ploneboard specific quesiton.

**Comment by merpdotcom** [comment]
I was hoping to find out if there were any plans to make Ploneboard work on Plone 5?

**Comment by sureshvv** [comment]
I don't think it is in anyone's front burner yet. Will be happy to help you with it if you have an urgent need.


________________________________
